barretenberg/cpp/src/barretenberg/vm/avm/tests/control_flow.test.cpp

```diff
@@ -11,31 +11,19 @@ using namespace bb::avm_trace;
 
 namespace {
 
-void validate_internal_call(Row const& row, uint32_t current_pc, uint32_t target_pc, uint32_t stack_ptr)
+void validate_internal_call(Row const& row, uint32_t current_pc, uint32_t target_pc)
 {
     EXPECT_EQ(row.main_sel_op_internal_call, FF(1));
     EXPECT_EQ(row.main_pc, FF(current_pc));
     EXPECT_EQ(row.main_ia, FF(target_pc));
-    EXPECT_EQ(row.main_internal_return_ptr, FF(stack_ptr));
-    EXPECT_EQ(row.main_sel_mem_op_b, FF(1));
-    EXPECT_EQ(row.main_rwb, FF(1));
     EXPECT_EQ(row.main_ib, FF(current_pc + Deserialization::get_pc_increment(OpCode::INTERNALCALL)));
-    EXPECT_EQ(row.main_mem_addr_b, FF(stack_ptr));
-    EXPECT_EQ(row.main_w_in_tag, FF(static_cast<uint32_t>(AvmMemoryTag::U32)));
-    EXPECT_EQ(row.main_space_id, FF(INTERNAL_CALL_SPACE_ID));
 };
 
-void validate_internal_return(Row const& row, uint32_t current_pc, uint32_t return_pc, uint32_t stack_ptr)
+void validate_internal_return(Row const& row, uint32_t current_pc, uint32_t return_pc)
 {
     EXPECT_EQ(row.main_sel_op_internal_return, FF(1));
     EXPECT_EQ(row.main_pc, FF(current_pc));
     EXPECT_EQ(row.main_ia, FF(return_pc));
-    EXPECT_EQ(row.main_internal_return_ptr, FF(stack_ptr));
-    EXPECT_EQ(row.main_sel_mem_op_a, FF(1));
-    EXPECT_EQ(row.main_rwa, FF(0));
-    EXPECT_EQ(row.main_mem_addr_a, FF(stack_ptr - 1));
-    EXPECT_EQ(row.main_r_in_tag, FF(static_cast<uint32_t>(AvmMemoryTag::U32)));
-    EXPECT_EQ(row.main_space_id, FF(INTERNAL_CALL_SPACE_ID));
 };
 
```

Comment:
ah - we no longer store use the `stack_ptr`. It was part of the old way where the internal stack was stored in "avm memory" and tracked with this pointer. We've since transitioned to a context specific stack (which is just a cpp stack so no `ptr`)